packages/openid4vc/src/openid4vc-issuer/router/authorizationChallengeEndpoint.ts

```diff
@@ -148,6 +142,70 @@ async function handleAuthorizationChallengeNoAuthSession(options: {
     )
   }
 
+  if (issuanceSession?.chainedIdentity?.externalAuthorizationServerUrl) {
+    const clientId =
+      parseResult.authorizationChallengeRequest.client_id ??
+      (parseResult.clientAttestation?.clientAttestationJwt
+        ? // FIXME: export scheme for client attestation jwt
+          // FIXME: add sub to base jwt type
+          decodeJwt({
+            jwt: parseResult.clientAttestation.clientAttestationJwt,
+          }).payload.sub
+        : undefined)
+
+    if (!clientId || typeof clientId !== 'string') {
+      throw new Oauth2ServerErrorResponseError({
+        error: Oauth2ErrorCodes.InvalidClient,
+        error_description: `Missing required 'client_id' parameter in authorization request.`,
+      })
+    }
+
+    const { pushedAuthorizationResponse } = await handlePushedAuthorizationRequest(agentContext, {
+      issuer,
+      issuanceSession,
+      parsedAuthorizationRequest: {
+        clientAttestation: parseResult.clientAttestation,
+        dpop: parseResult.dpop,
+        authorizationRequest: {
+          ...parseResult.authorizationChallengeRequest,
+          response_type: 'code',
+          client_id: clientId,
+        },
+      },
+      request,
+    })
+
+    throw new Oauth2ServerErrorResponseError({
+      error: Oauth2ErrorCodes.RedirectToWeb,
+      error_description: 'Chained identity required before issuance',
```

Comment:
To the wallet it's just important that an authorization is required before issuance.

```suggestion
      error_description: 'Authorization required before issuance',
```